src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -206,7 +257,8 @@ function getItems({
   return items.reduce<any[]>(
```

Comment:
You can pass `items.reduce<CurrentRefinementsConnectorParamsItem[]>` to have it fully typed below (in `filter` and `refine`).

Reply:
Done.